src/xcode/ENA/ENA/Source/Services/HealthCertificate/__test__/HealthCertificateServiceTests.swift

```diff
@@ -530,7 +536,8 @@ class HealthCertificateServiceTests: CWATestCase {
 
 		let service = HealthCertificateService(
 			store: store,
-			signatureVerifying: DCCSignatureVerifyingStub(error: .HC_COSE_NO_SIGN1),
```

Comment:
why is the test still succeeding when you do not pass an error into it? Is there a logical mistake in the test?

Reply:
Same here 😅